src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyPage.tsx

```diff
@@ -89,6 +89,34 @@ const MethodologyPage: NextPage<Props> = ({ data }) => {
         </div>
         <div className="govuk-grid-column-one-third">
           <RelatedInformation>
+            {data.publications && data.publications.length > 0 && (
```

Comment:
We might want to use optional chaining here instead of the two `&&` to be more concise. Not sure if we do this in other places. In addition to this, should we be returning some generic text just in case we have no publications? i.e. 
```
{data?.publications?.length > 0  ? (
  // display the publications
) : ( 
<p>No publications available</p>
) 

}
```

Reply:
Thanks @luke-h1 just made a slight change to use optional chaining for the notes, publications and annexes properties. Never expecting `data` to be null. Also I didn't include any default text as this shouldn't occur and I think it's worse for a general public user to see `No publications available` which they don't understand than not see the Publications heading at all.